src/minimum_dependencies/_core.py

```diff
@@ -86,11 +85,13 @@ def minimum_version(requirement: Requirement, fail: Fail = Fail.FALSE) -> Versio
 
     # If the specified version does not exist on PyPi, issue a warning
     # and return the lowest available version
-    msg = f"Exact version specified in {requirement} not found on PyPi."
+    msg = f'Could not find "{requirement}" on PyPi'
```

Comment:
```suggestion
    msg = f'Could not find {requirement} on PyPi'
```

Again not a fan of quotes. Use "`" marks if you must.